doc/radosgw/layout.rst

```diff
@@ -38,40 +39,40 @@ You can use the following commands to introspect metadata entries: ::
     
 Some variables have been used in above commands, they are:
 
-- user: Holds user information
-- bucket: Holds a mapping between bucket name and bucket instance id
-- bucket.instance: Holds bucket instance information[2]
+- _user_: Holds user information
```

Comment:
why add underscores? are there underscores around the actual tokens?

Reply:
That's meant to italicize in the output.

![Screenshot 2025-02-04 at 11 48 45 AM](https://github.com/user-attachments/assets/704db0e0-7433-4fea-ab16-95f5dd6f27c2)